### ISSUE TITLE
report timeouts 

### ISSUE THREAD

**Issue by wholmgren** [SolarArbiter/solarforecastarbiter-core#526]
I've been waiting ~10 minutes for a 14 day, 2 forecast (hourly intervals) report to compute. It has just a few metrics and categories, so I expected it would finish in just a few seconds.

https://dashboard.solarforecastarbiter.org/reports/952ef89c-c641-11ea-a5d6-0a580a820020

**Comment by alorenzo175** [comment]
Yeah, an unfortunate error as the worker tried to start the job so it never got posted that it failed. I think we want to investigate this specific error more, but not put too much effort into ensuring some kind of message gets back to the database. Instead, maybe worth updating the pending text to say when it was started and a recommendation to recompute after x amount of time. 

![image](https://user-images.githubusercontent.com/4625457/87563386-0031ef80-c674-11ea-99e8-9852baf2f91c.png)

**Comment by wholmgren** [comment]
Hm, I thought I tried to hit recompute at some point but could be wrong.

**Comment by wholmgren** [comment]
This seems to have occurred again with report https://dashboard.solarforecastarbiter.org/reports/b67fd8e2-c70a-11ea-b944-0a580a820032 appeared to require two attempts to recompute
